steps/reasons-for-appealing/reason-for-appealing/ReasonForAppealing.js

```diff
@@ -1,40 +1,64 @@
 'use strict';
 
-const { Question, goTo } = require('@hmcts/one-per-page');
-const { form, textField } = require('@hmcts/one-per-page/forms');
+const { AddAnother } = require('@hmcts/one-per-page/steps');
+const { goTo } = require('@hmcts/one-per-page');
+const { text, object } = require('@hmcts/one-per-page/forms');
 const { answer } = require('@hmcts/one-per-page/checkYourAnswers');
-const { whitelist } = require('utils/regex');
 const sections = require('steps/check-your-appeal/sections');
+const content = require('steps/reasons-for-appealing/reason-for-appealing/content.en');
+const { errorFor } = require('@hmcts/one-per-page/src/forms/validator');
```

Comment:
Can you move this up with the others so we group all `{ dependencies }` together.